contentful_management/client.py

```diff
@@ -317,7 +317,7 @@ def editor_interfaces(self, space_id, content_type_id):
 
         return EditorInterfacesProxy(self, space_id, content_type_id)
 
-    def snapshots(self, space_id, entry_id):
+    def snapshots(self, space_id, resource_id, resource_kind='entries'):
```

Comment:
I am not sure if I understand your reply. I was refering to the possibility to have the `snapshots` as part of the `entry`-proxy and not as a separate proxy, as it is the case right now ....